### PR TITLE
Fix for smoke tests

### PR DESCRIPTION
**Pull request by yrong**
- Seems `Utility::batch_all` through xcm is not allowed in bridgeHub which we depend on for some of the smoke tests, some revamp accordingly
- Xcm version of Ethereum should be configured in assetHub for `reserve_transfer`

### CODE REVIEW

File: smoketest/src/helper.rs [hunk 1]
```diff
@@ -286,7 +286,7 @@ pub async fn construct_transfer_native_from_agent_call(
 	Ok(call)
 }
 
-pub async fn governance_bridgehub_call_from_relay_chain(
+pub async fn governance_bridgehub_batch_call_from_relay_chain(
```
Comment (claravanstaden) [inline smoketest/src/helper.rs:289]: I guess we can remove this if it is not used anymore?

**Review by claravanstaden — APPROVED** [review]
Nice! 👏🏻

**Review by vgeddes — APPROVED** [review]
+1

**Review by vgeddes — APPROVED** [review]
+1

**Pull request merged**